src/sentry/tasks/sentry_apps.py

```diff
@@ -217,6 +220,17 @@ def _process_resource_change(action, sender, instance_id, retryer=None, *args, *
         # Trigger a new task for each webhook
         send_resource_change_webhook.delay(installation_id=installation.id, event=event, data=data)
 
+    if features.has("organizations:sentry-functions", org):
```

Comment:
@vpandiarajan20 To be safe, let's make sure we don't even handle the case where `instance` is not an instance of `Group`. if `instance` is `Event`, we should just end early. We only want to handle the issues case and this function does issues and errors.